sdv/data_processing/data_processor.py

```diff
@@ -751,7 +751,17 @@ def reverse_transform(self, data, reset_keys=False):
             if is_integer_dtype(dtype) and is_float_dtype(column_data.dtype):
                 column_data = column_data.round()
 
-            reversed_data[column_name] = column_data[column_data.notna()].astype(dtype)
+            try:
+                reversed_data[column_name] = column_data[column_data.notna()].astype(dtype)
+            except ValueError:
```

Comment:
Is it possible that this crashes on a column that is not pii, and if so, do we still want to catch the error?